docs/.vuepress/components/cdr-doc-intro.vue

```diff
@@ -60,7 +60,7 @@ export default {
   }
 
   .cdr-doc-intro--small {
-    padding: $space-2-x $space-1-x;
+    padding: 52px $space-1-x;
```

Comment:
why the magic number?

Reply:
https://issues.rei.com/browse/PLIB-3749

"Container height should be 188px in size total"

Reply:
Again, let's use the system. Can we update the 52px to $space-4-x for article headers and $space-2-x for component headers?